udata/forms/__init__.py

```diff
@@ -8,7 +8,7 @@
 from flask_mongoengine.wtf import model_form  # noqa
 
 from flask_mongoengine.wtf.models import ModelForm as MEModelForm  # noqa
-from flask_wtf import Form as WTForm  # noqa
+from flask_wtf import FlaskForm  # noqa
```

Comment:
I was happy too to remove it but sadly this is here because of the mandatory early `wtforms_json.init()`